bindings/node/index.d.ts

```diff
@@ -131,6 +131,45 @@ export interface KMSProviders {
   }
 }
 
+/**
+ * TLS options to use when connecting. The spec specifically calls out which insecure
+ * tls options are not allowed:
+ *
+ *  - tlsAllowInvalidCertificates
+ *  - tlsAllowInvalidHostnames
+ *  - tlsInsecure
+ *  - tlsDisableOCSPEndpointCheck
+ *  - tlsDisableCertificateRevocationCheck
+ */
+export interface ClientEncryptionTLSOptions {
+  /**
+   * Enables or disables TLS/SSL for the connection.
+   */
+  tls?: boolean;
+  /**
+   * Specifies the location of a local TLS Certificate
+   */
+  tlsCertificateFile?: string;
```

Comment:
In [NODE-3591](https://jira.mongodb.org/browse/NODE-3591) we talked about deprecating this – my understanding is that this is only being added for a new feature (KMIP support), so maybe it’s a good idea to omit this?

Reply:
Thanks for the suggestion - I've removed.